altofocus/assets/js/jquery.flexslider.js

```diff
@@ -217,14 +217,33 @@
           if (slider.pagingCount > 1) {
             for (var i = 0; i < slider.pagingCount; i++) {
               slide = slider.slides.eq(i);
-              if ( undefined === slide.attr( 'data-thumb-alt' ) ) { slide.attr( 'data-thumb-alt', '' ); }
-              var altText = ( '' !== slide.attr( 'data-thumb-alt' ) ) ? altText = ' alt="' + slide.attr( 'data-thumb-alt' ) + '"' : '';
-              item = (slider.vars.controlNav === "thumbnails") ? '<img src="' + slide.attr( 'data-thumb' ) + '"' + altText + '/>' : '<a href="#">' + j + '</a>';
+
```

Comment:
This is an external library, I'm not sure why there are any differences, but they look like whitespace changes to me.

Reply:
Looks like an SVN-only change: https://opengrok.a8c.com/source/history/pub/altofocus/assets/js/jquery.flexslider.js

Bringing this change in.